meteor/server/api/peripheralDevice.ts

```diff
@@ -1150,5 +1152,16 @@ class ServerPeripheralDeviceAPIClass extends MethodContextAPI implements NewPeri
 	) {
 		await PackageManagerIntegration.removePackageInfo(this, deviceId, deviceToken, type, packageId)
 	}
+	// --- Triggers ---
+	async inputDeviceTrigger(
```

Comment:
As a first-time-reader, this is not immediately clear when or why it is called.
Perhaps add a description here?
```
/**
This method is called when a button is pressed, a knob is turned etc..
@param value will be a boolean when a button is pressed (?), a number when a knob is turned (?)...

 */
```

(I know this is not documented on any of the other methods, but still....)

Reply:
I agree, but should this be documented here (where this is an implementation of an interface) or should it be in `@sofie-automation\shared-lib\src\peripheralDevice\methodsAPI.ts`, where this documentation will be available to consumers? I think I'm going to go for the latter.